assets/js/components/ClusterDetails/ClusterSettings.test.jsx

```diff
@@ -42,4 +42,84 @@ describe('ClusterDetails ClusterSettings component', () => {
       screen.getByRole('button', { name: 'Select Checks for Execution' })
     ).toBeVisible();
   });
+
+  it('given VMware provider, should render the warning banner', async () => {
+    const group = faker.animal.cat();
+    const catalog = catalogCheckFactory.buildList(2, { group });
+
+    const [StatefulChecksSettings, state] = withState(<ClusterSettings />, {
+      ...defaultInitialState,
+      catalog: { loading: false, data: catalog, error: null },
+      clustersList: {
+        clusters: defaultInitialState.clustersList.clusters.map((cluster) => ({
+          ...cluster,
+          provider: 'vmware',
+        })),
+      },
+    });
+
+    const {
```

Comment:
You don't need this `getState`. You should create the cluster with a factory and get the id from it